bsp/Infineon/libraries/HAL_Drivers/drv_adc.c

```diff
@@ -57,7 +57,7 @@ static rt_err_t ifx_adc_enabled(struct rt_adc_device *device, rt_uint32_t channe
 
         if (result != RT_EOK)
         {
-            LOG_E("ADC initialization failed. Error: %llu\n", (long unsigned int)result);
+            LOG_E("ADC initialization failed. Error: %lu\n", (long unsigned int)result);
```

Comment:
cy_rslt_t  应该是个u32位类型吧（麻烦确认一下cy_rslt_t 是什么类型），这里应该是没有必要做强制转换的
```suggestion
            LOG_E("ADC initialization failed. Error: %u\n", result);
```

Reply:
确定是U32 可以不用类型转换